src/test/mir-opt/array-index-is-temporary/32bit/rustc.main.SimplifyCfg-elaborate-drops.after.mir

```diff
@@ -44,23 +44,23 @@ fn main() -> () {
                                          // mir::Constant
                                          // + span: $DIR/array-index-is-temporary.rs:13:26: 13:28
                                          // + literal: Const { ty: u32, val: Value(Scalar(0x0000002c)) }
-        StorageLive(_2);                 // bb0[2]: scope 1 at $DIR/array-index-is-temporary.rs:14:9: 14:14
-        _2 = const 1usize;               // bb0[3]: scope 1 at $DIR/array-index-is-temporary.rs:14:17: 14:18
+        StorageLive(_2);
+        _2 = const 1usize;
```

Comment:
Hmm, I still want the scope/file/line/column part of the comment, the only thing that bothers me is the `bbX[Y]` because *that* changes with statements being removed by optimizations.